pkg/controller/user-cluster-controller-manager/resources/resources/mla/promtail/daemonset.go

```diff
@@ -90,21 +86,7 @@ func DaemonSetCreator(overrides *corev1.ResourceRequirements, registryWithOverwr
 					Type: corev1.SeccompProfileTypeRuntimeDefault,
 				},
 			}
-			ds.Spec.Template.Spec.InitContainers = []corev1.Container{
-				{
-					Name:            "init-inotify",
-					Image:           fmt.Sprintf("%s/%s:%s", registryWithOverwrite(resources.RegistryDocker), initImageName, initImageTag),
-					ImagePullPolicy: corev1.PullAlways,
-					Command: []string{
-						"sh",
-						"-c",
-						fmt.Sprintf("sysctl -w fs.inotify.max_user_instances=%d", inotifyMaxUserInstances),
-					},
-					SecurityContext: &corev1.SecurityContext{
-						Privileged: pointer.BoolPtr(true),
-					},
-				},
-			}
+			ds.Spec.Template.Spec.InitContainers = []corev1.Container{}
```

Comment:
is this required at all if we don't set up the initcontainer?

Reply:
I believe if we don't set it, the reconciling framework will use the previous values, so it will not remove the init container from existing DaemonSets.

Reply:
gotchu, makes sense!